Mage.Sets/src/mage/cards/l/LiberatedLivestock.java

```diff
@@ -0,0 +1,121 @@
+package mage.cards.l;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.DiesSourceTriggeredAbility;
+import mage.abilities.effects.OneShotEffect;
+import mage.cards.*;
+import mage.constants.Outcome;
+import mage.constants.SubType;
+import mage.constants.CardType;
+import mage.constants.Zone;
+import mage.filter.FilterCard;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.game.permanent.token.*;
+import mage.players.Player;
+import mage.target.TargetCard;
+import mage.target.common.TargetCardInHand;
+import mage.target.common.TargetCardInYourGraveyard;
+
+/**
+ *
+ * @author Codermann63
+ */
+public final class LiberatedLivestock extends CardImpl {
+
+    public LiberatedLivestock(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{5}{W}");
+        
+        this.subtype.add(SubType.CAT);
+        this.subtype.add(SubType.BIRD);
+        this.subtype.add(SubType.OX);
+        this.power = new MageInt(4);
+        this.toughness = new MageInt(6);
+
+
+
+        // When Liberated Livestock dies, create a 1/1 white Cat creature token with lifelink,
+        // a 1/1 white Bird creature token with flying,
+        // and a 2/4 white Ox creature token.
+        // For each of those tokens, you may put an Aura card from your hand and/or graveyard onto the battlefield attached to it.
+        this.addAbility(new DiesSourceTriggeredAbility(new LiberatedLivestockEffect()));
+    }
+
+    private LiberatedLivestock(final LiberatedLivestock card) {
+        super(card);
+    }
+
+    @Override
+    public LiberatedLivestock copy() {
+        return new LiberatedLivestock(this);
+    }
+}
+
+class LiberatedLivestockEffect extends OneShotEffect {
+
+    LiberatedLivestockEffect() {
+        super(Outcome.PutCreatureInPlay);
+        staticText = "create a 1/1 white Cat creature token with lifelink, a 1/1 white Bird creature token with flying, and a 2/4 white Ox creature token. For each of those tokens, you may put an Aura card from your hand and/or graveyard onto the battlefield attached to it.";
+    }
+
+    private LiberatedLivestockEffect(final LiberatedLivestockEffect effect) {super(effect);}
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Player controller = game.getPlayer((source.getControllerId()));
+        if (controller == null){
+            return false;
+        }
+        Token cat = new CatToken2();
+        Token bird = new BirdToken();
+        Token ox = new OxToken();
+
+        // pause to get token onto the battlefield all the same time
+        game.getState().pause();
```

Comment:
Well that's an inner method, this is not a normal technique. Is there a bug that occurs without the pause?

Reply:
I believe there is a but with Mystic Reflection without the pause:

Mystic Reflection {1}{U}
Instant
Choose target nonlegendary creature. The next time one or more creatures or planeswalkers enter the battlefield this turn, they enter as copies of the chosen creature.
Foretell {U}

Without the pause only the cat token enters as a copy, not all three. If you have a solution I'm all ears :)

Reply:
I do think the usual way to ensure continuous effects are applied is to call `game.getState().processAction(game)`.

Reply:
I fiddled with it again and it works without pause & unpause.